MAUI/Calendar/views.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: Views in MAUI Calendar widget | Syncfusion
+description: Learn here about the Views feature of Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: Calendar
+documentation: ug
+---
+
+
+# Multiple Calendar Views in MAUI (SfCalendar)
+The `SfCalendar` widget has four Calendar views to display. It can be assigned to the widget by using the `View` property. `Month` view is initially rendered by Default. The current date will be displayed initially for all the Calendar views.
+
+## Month view
+The Month view displays the current month days, and usually a few days of previous and next month. By default, initially displays the current month dates and the current date is highlighted by a separate color different from the rest of the dates color in `Month` view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+{% include_relative code-snippet/month-view.xaml %}
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.View = CalendarView.Month;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view in .NET MAUI Calendar.](images/views/maui-month-view.png)
+
+### Number Of Visible Weeks view
+In the month view, number of visible weeks can be customized by using the `NumberOfVisibleWeeks` property in the Calendar. By default, Month view displays with the NumberOfVisibleWeeks as `6`.
+
+The following code shows the Calendar month view with `NumberOfVisibleWeeks` is `4`.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar"
+                      View="Month">
+                      <Calendar:SfCalendar.MonthView>
+                        <Calendar:CalendarMonthView NumberOfVisibleWeeks = 4/>
+                      </Calendar:SfCalendar.MonthView>                    
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView = new CalendarMonthView()
+{
+    NumberOfVisibleWeeks = 4,
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![custom-number-of-weeks-in-maui-calendar](images/views/custom-number-of-weeks-in-maui-calendar.png)
+
+### Week number
+It displays week number for the current view dates in the month view by setting the `ShowWeekNumber` property. By default, the ShowWeekNumber is `false`. If you need to show the week number in the month view by setting the ShowWeekNumber as true. Week numbers will be displayed based on the ISO standard.
```

Comment:
By setting the `ShowWeekNumber` property, it displays the week number for the current view dates in the month view. By default, the `ShowWeekNumber` is set to `false.` If you need to show the week number in the month view, set the `ShowWeekNumber` as true. The week numbers will be displayed based on the ISO standard.

Reply:
Week number content changed